src/pages/Home.js

```diff
@@ -1,72 +1,172 @@
-          hr({ className: 'home-line' }),
-          div({ style: { textAlign: 'center' } }, [
-            div({ isRendered: !this.props.isLogged }, [
-              div({ style: { fontSize: 'larger', fontWeight: '500' } }, ['Want to explore datasets and make a data access request?']),
-              div({ style: { margin: '1rem 0 1rem 0' } }, [
-                SignIn({ props: this.props, onSignIn: () => this.props.onSignIn(), history: this.props.history })
-              ]),
-              div({ style: { margin: '0 0 1rem 0' } }, [
-                span({}, [
-                  'Don\'t have a Google Account? Create a new one ',
-                  a({
-                    href: 'https://accounts.google.com/SignUp?continue:https%3A%2F%2Faccounts.google.com%2Fo%2Foauth2%2Fauth%3Fopenid.realm%26scope%3Demail%2Bprofile%2Bopenid%26response_type%3Dpermission%26redirect_uri%3Dstoragerelay%3A%2F%2Fhttp%2Flocalhost%3A8000%3Fid%253Dauth721210%26ss_domain%3Dhttp%3A%2F%2Flocalhost%3A8000%26client_id%3D832251491634-smgc3b2pogqer1mmdrd3hrqic3leof3p.apps.googleusercontent.com%26fetch_basic_profile%3Dtrue%26hl%3Des-419%26from_login%3D1%26as%3D43c5de35a7316d00&ltmpl:popup',
-                    target: '_blank'
-                  }, ['here, ']),
-                  'or sign up with an existing email address ',
-                  a({
-                    href: 'https://support.google.com/accounts/answer/176347',
-                    target: '_blank'
-                  }, ['using these instructions.'])
-                ])
-              ])
+      div({ className: 'row' }, [
+        div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12' }, [
+          div({ className: 'row no-margin', style: { backgroundColor: 'white', height: '350px', position: 'relative' }}, [
+            img({ style: { height: 'inherit', minWidth: '100%', transform: 'scale(1.1,1)' }, src: '/images/home_header_background.png'}),
+            div({ style: { position: 'absolute', width: '100%', top: '50%', left: '50%', transform: 'translate(-50%, -50%)'}}, [
+              img({ style: duosLogo , alt: 'DUOS logo', src: '/images/duos_logo.svg' }),
+              h1({ style: homeTitle }, ['Data Use Oversight System']),
+              div({ style: homeBannerDescription }, [
+                'Expediting data access for researchers, by facilitating and \nenhancing data access committee\'s workflows'])
             ]),
-            p({ style: { fontSize: 'larger', fontWeight: '500' } }, ['Are you a DAC member who wants to learn more about DUOS?']),
-            p({ style: { fontSize: 'large', fontWeight: '350' } }, ['Email us! ']),
-            p({ style: { fontSize: 'large', fontWeight: '500' } }, [
-              h(Mailto, { email: 'DUOS-support@broadinstitute.zendesk.com' })
+            div({ isRendered: !this.props.isLogged, style: { padding: '2em 2em 0 0', display: 'flex', alignItems: 'center', position: 'absolute', top: '1rem', right: '1rem'}}, [
```

Comment:
One of the suggestions I made for Justin's PR is that we try to destructure as many props at the top of the render method into constants. That way, anyone who works on this code will know all of the props that are required in one place. If they're spread out through the html like this, some can be easily overlooked.

Reply:
I believe I've addressed this now (wasn't sure if I needed to handle `props: this.props`, but destructured the rest), if you could check it again.